Ingestion/templates/datahub-executor.ecs.template.yaml

```diff
@@ -0,0 +1,668 @@
+        DatahubIngestionsSignalPollInterval:
+          Default: "Ingestion queue signal polling interval. Warning - do not change this without consulting with your Acryl rep."
+        ExistingDataHubAccessTokenSecretArn:
+          Default: "Existing DataHub Access Token Secret Arn, if not specified, will use DataHubAccessToken"
+        OptionalSecrets:
+          Default: ""
+        OptionalEnvVars:
+          Default: "Optional environment variables to set in the ECS task container."
+        DesiredCount:
+          Default: "Number of worker replicas"
+        ImageTag:
+          Default: "The image tag for the Acryl Remote Executor, defaults to the latest release."
+        TaskCpu:
+          Default: "The ECS Task CPU value, check https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html"
+        TaskMemory:
+          Default: "The ECS Task Memory value, check https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html"
+        TaskEphemeralStorageSizeInGiB:
+          Default: "The amount of ephemeral storage to allocate for the task"
+Parameters:
+  SubnetID:
+    Type: "AWS::EC2::Subnet::Id"
+  VPCID:
+    Type: "AWS::EC2::VPC::Id"
+  DataHubBaseUrl:
+    Type: "String"
+    Default: "https://xxx.acryl.io/gms"
+    Description: "DataHub Base Url, for example: https://xxx.acryl.io/gms"
+  DataHubAccessToken:
+    Type: "String"
+    NoEcho: true
+    Default: ""
+    Description: "DataHub Personal Access Token. If not specified, we'll get it from ExistingDataHubAccessTokenSecretArn"
+  ExecutorId:
+    Type: "String"
+    Description: "Unique Executor Id. Warning - do not change this without consulting with your Acryl rep."
+    Default: "remote"
+  DatahubIngestionsMaxWorkers:
+    Type: "String"
+    Description: "Maximum number of ingestion tasks allowed to run in parallel. Warning - do not change this without consulting with your Acryl rep."
+    Default: ""
+  DatahubMonitorsMaxWorkers:
+    Type: "String"
+    Description: "Maximum number of monitoring tasks allowed to run in parallel. Warning - do not change this without consulting with your Acryl rep."
+    Default: ""
+  DatahubIngestionsSignalPollInterval:
+    Type: "String"
+    Description: "Ingestion queue signal polling interval. Warning - do not change this without consulting with your Acryl rep."
+    Default: ""
+  ExistingDataHubAccessTokenSecretArn:
+    Type: "String"
+    Default: ""
+    Description: "Existing DataHub Access Token Secret Arn, if not specified, will use DataHubAccessToken"
+  AwsRegion:
+    Type: "String"
+    Default: "us-west-2"
+    Description: "AWS Region where executor is hosted."
+  OptionalSecrets:
+    Type: "String"
+    Default: ""
+    Description: "A List of Optional AWS Secrets,for example: SECRET01_ENV=arn:aws:secretsmanager:us-east-1:111111111111:secret:xxx,SECRET02_ENV=arn:aws:secretsmanager:us-east-1:111111111111:secret:yyy (NO SPACE, supports up to 10 secrets)"
```

Comment:
Let's add: The name of these secrets should match the name of the secrets in your DataHub recipe.

Reply:
added